Assets/Scripts/EnemyAI.cs

```diff
@@ -0,0 +1,25 @@
+﻿using UnityEngine;
+using UnityEngine.AI;
+
+public class EnemyAI : MonoBehaviour
+{
+    //private GameObject enemy;
+    public Transform[] waypoints;
+    private int m_Index=0;
```

Comment:
Don't use prefixes for variables. Consistency in the code is important. Run autoformat. This should be `index = 0` with spaces